backend/src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
+spring.datasource.url = jdbc:postgresql://localhost:3306/notes_app?useSSL=false
+spring.datasource.username = root
+spring.datasource.password = root
 
+spring.jpa.hibernate.ddl-auto = validate
```

Comment:
Please don't push this file in the pr because everyone has it's own dev envinronment